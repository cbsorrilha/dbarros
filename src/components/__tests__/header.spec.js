import React from "react"
import renderer from "react-test-renderer"
import Header from '../header';


describe('Snapshot testing the Header', () => {
    it("renders correctly", () => {
        const props = {}
        const tree = renderer.create(<Header {...props} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
