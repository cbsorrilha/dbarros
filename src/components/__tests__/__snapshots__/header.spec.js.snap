// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing the Header renders correctly 1`] = `
<header
  className="sc-AykKC fLWTSL"
/>
`;
